_topics/en/version-3-transaction-relay.md

```diff
@@ -46,20 +46,24 @@ see_also:
 
   - title: Anchor outputs
     link: topic anchor outputs
+---
+**Version 3 transaction relay** is a proposal to allow transactions to
+opt-in to a modified set of transaction relay policies designed to prevent
+pinning attacks. Combined with package relay, these policies help enable
+the use of dynamic feerates with LN onchain transactions.
+ 
+V3 transaction relay is a superset of standard transaction policy.
+That is, V3 transactions follow all rules for standard transactions
+(e.g. minimum and max transaction weight) while also adding some 
+non-standard rules designed to allow transaction replacement 
+while precluding transaction-pinning attacks. V3 transactions also 
+require minor changes to the Package RBF policy in order to maintain 
+incentive compatibility with miners.
 
-## Optional.  Force the display (true) or non-display (false) of stub
-## topic notice.  Default is to display if the page.content is below a
-## threshold word count
-#stub: false
+V3 transaction relay solves rule 3 [transaction pinning][topic version 3 transaction relay]
+and may allow the removal of the CPFP carve-out.
 
-## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
-## Should be less than 500 characters
-excerpt: >
```

Comment:
The excerpt field is required, so you'll need to move the first paragraph of the text back into the excerpt.  (Jekyll automatically generates an excerpt for pages from the first paragraph, but we need the excerpt field to be defined in the YAML in order to perform some additional validation on it, such as ensuring that it is less than 500 characters.)

Reply:
Fixed & squashed. git is tricky :)